### DISCUSSION TITLE
Potential for a Windows support team

### DISCUSSION THREAD

**Discussion by kurtferreira** [zed-industries/zed#6618] [Feature Requests]
Since the platform support discussion is locked, thought I'd pop this question here for the Zed team not to cause frustration but assistance.

I'm aware the current MO is to further the functionality of the Zed editor (which is great btw) and to focus on issues surrounding the core, but is there no potential for a Windows collab team (volunteer basis) to assist with moving the project along on that platform?

My own motivation is quite simple: I used Mac at the office, Windows at home, so I feel the true value Zed offers during development and it pains me that I'm not able to use it on Windows yet. Secondly, I'm sure that opening it up to another platform will likely expose some other issues related to the core which we wouldn't wan to burden the core team with, but if the Windows team was focused purely on the platform-specific support, what a win.

**Comment by jansol** [comment]
~~I don't think there would be anything or anyone stopping such a team from forming once the code is public but at the same time I can't really see it happening before that, especially not as a volunteer effort (the lack of source access makes any kind of volunteer work kind of difficult).~~

**Comment by JosephTLyons** [comment]
At the moment, we are simply sprinting towards open source and likely won't be able to do anything with platform support until after. But, we are getting closer and closer every day. Once we are there, we certainly would love help from the community in getting windows/linux support up and running.

**Reply by JenuelDev** [reply]
Nise, there are a ton of programmers using windows OS. We are excited to use zed.

**Reply by TatryProject** [reply]
I'd like to throw my hat in this ring. I'd love to volunteer to provide assistance toward a Windows or Linux release. I have a background in C/C++, though I am not too familiar with Rust, but am more than happy to learn prior or by doing. That said, in the meantime if there are any resources the team or anyone recommends for picking up Rust, I'd appreciate that as well

**Reply by Dishpit** [reply]
Just wanna chime in and say: as a dev that uses all three OSes on a regular basis I appreciate the community coming together for the effort of bringing Windows and Linux support 😭

**Comment by enp6** [comment]
Do you know why the Windows platform is not supported? Because it simply can't compete with the editors on the Windows platform, nor does it have any experience in optimizing. Because Mac has a smaller user base, they are more inclined to develop small platforms to obtain new investments. I don't think anyone needs to wait for this.

**Reply by wistfuljoker** [reply]
which are those editors that exist only in windows and zed can't compete?

**Reply by dredstone1** [reply]
> which are those editors that exist only in windows and zed can't compete?

he means the slow vscode and the expensive jetbrain products....

**Comment by tml** [comment]
Discussion about platform support (Windows / Linux) can be found at https://github.com/zed-industries/zed/issues/5391
